src/SFA.DAS.QnA.Application/Commands/SetPageAnswers/SetAnswersBase.cs

```diff
@@ -120,10 +120,28 @@ protected Next GetNextAction(Page page, List<Answer> answers, ApplicationSection
                             var applicationData = JObject.Parse(application.ApplicationData);
                             var questionTag = applicationData[condition.QuestionTag];
 
-                            if (questionTag == null || questionTag.Value<string>() != condition.MustEqual)
+
+                            if (questionTag == null)
                             {
                                 someConditionsNotSatisfied = true;
                             }
+                            else
+                            {
+                                if (!string.IsNullOrEmpty(condition.MustEqual) && questionTag.Value<string>() != condition.MustEqual)
```

Comment:
You might need to look at line 65 and do something similar there too: GetCheckboxListMatchingNextActions

Reply:
I have now made that change